### PR TITLE
CHG Add/Assign Questions Button Color Issue #320

### PR DESCRIPTION
**Pull request by AaronBinChen**

### CODE REVIEW

File: web/src/components/core/Assignment/QuestionControls.jsx [hunk 1]
```diff
@@ -148,15 +153,20 @@ export default function QuestionControls({assignmentId, reload, questionsAssigne
             {/* Add question button */}
             <Grid item xs={12} sm={6} md={3}>
               <Tooltip title={'Add question'}>
+                {/* <ThemeProvider theme={theme}> */}
                 <Button
+                  // style = {{
```
Comment (synoet) [inline web/src/components/core/Assignment/QuestionControls.jsx:158]: Please remove your comments

Comment (AaronBinChen) [reply web/src/components/core/Assignment/QuestionControls.jsx]: removed

**Review by synoet — APPROVED** [review]
LGTM!

**Review by synoet — CHANGES_REQUESTED** [review]
Sorry that approval was a little preemptive. Please sign and squash your commits.

**Pull request closed without merging**